diff --git a/build.sh b/build.sh
index 37588d1..788dc60 100755
--- a/build.sh
+++ b/build.sh
@@ -13,7 +13,7 @@ mkdir build

 pushd build &>/dev/null

-CMAKE_EXTRA_ARGS=""
+: ${CMAKE_EXTRA_ARGS}

 if [ $OPERATING_SYSTEM == "Android" ]; then
   export CC="$CC -D__ANDROID_API__=26"
@@ -68,6 +68,8 @@ cmake ../src/llvm \
  -DTAPI_FULL_VERSION=$TAPI_VERSION \
  $CMAKE_EXTRA_ARGS

+if [ -z "$NOMAKE" ]; then
+
 echo ""
 echo "## Building clangBasic ##"
 echo ""
@@ -80,5 +82,7 @@ echo ""

 $MAKE libtapi -j $JOBS

+fi
+
 popd &>/dev/null
 popd &>/dev/null
